test/setupTests.ts

```diff
@@ -46,17 +45,6 @@ global.crypto = {
   },
 };
 
-//sentry testing was causing OOM for some reason
-//const { testkit, sentryTransport } = sentryTestkit();
-//global.testKit = testkit;
-
-beforeAll(() => {
-  /*Sentry.init({
-    dsn: "https://testKey@o782870.ingest.sentry.io/0",
-    transport: sentryTransport,
-  });*/
-});
-
```

Comment:
I've moved this code to the Button comp test, which is the only place it is used.

Reply:
With it in the global setup, it causes problems for the Datepicker test (which was already commented out). Here is the error created when it is configured in global: 
`  ● DatePicker › typing works in zh-TW (same for all languages/timezones)

    TestingLibraryElementError: Unable to find an accessible element with the role "textbox"`
I can't work out how to fix this without getting rid of Sentry or configuring it locally in the Button comp.

Reply:
I've left sentry and the button test commented out as it is in master